websrc/source/documents/tutorials/prometheus-tutorial.md

```diff
@@ -0,0 +1,145 @@
+
+Now let's move to Grafana. 
+
+Grafana is exposed at port 32701, so let's navigate to http://192.168.2.54:32701 in our browser. The Grafana UI will open up and prompt you for a username and password. Username is "admin" and password is "admin". We will be working with Grafana for the remainder of this tutorial because it's UI is more user friendly.
+
+After you have logged in, you will see an option to `Add data source`. Click on that. Fill out the information as show in the following screenshot.
+
+![Add Data Source](add_data_source.png)
+
+Click on `Add` and `Save & Test`.
+
+Click on the Grafana logo in the top left of the screen and go to Dashboard and click on `New`.
+Click on `Graph`. Click on "Panel Title" and then click on "Edit".
+
+### <a name="ch2"></a> Chapter 2: Basic Ping Example
+
+Now go back to your terminal which has the cluster running. You should already have a `contiv-net` setup as part of the Networking Tutorial as well as the `contiv-c1` and `contiv-c2` pods up and running.
+
+```
+[vagrant@kubeadm-master ~]$ netctl net ls
+Tenant   Network      Nw Type  Encap type  Packet tag  Subnet        Gateway    IPv6Subnet  IPv6Gateway  Cfgd Tag
+------   -------      -------  ----------  ----------  -------       ------     ----------  -----------  ---------
+default  default-net  data     vxlan       0           20.1.1.0/24   20.1.1.1
+default  contiv-net   data     vxlan       0           10.1.2.0/24   10.1.2.1
+default  contivh1     infra    vxlan       0           132.1.1.0/24  132.1.1.1
+...
+
+
+[vagrant@kubeadm-master ~]$ kubectl get pods
+NAME        READY     STATUS    RESTARTS   AGE
+contiv-c1   1/1       Running   0          24s
+contiv-c2   1/1       Running   0          9s
+...
+```
+
+Now let's go back to the Grafana UI and add the following four queries.
+
+```
+rx_packets{containerName="contiv-c1"}
+rx_packets{containerName="contiv-c2"}
+tx_packets{containerName="contiv-c1"}
+tx_packets{containerName="contiv-c2"}
+```
+
+Go back to the terminal and set up a ping between the two pods.
+
+```
+[vagrant@kubeadm-master ~]$ kubectl exec -it contiv-c1 sh
+/ # ping 10.1.2.3
+```
+Now if we go back to the Grafana Dashboard, we should see that the number of packets transmitted increases, so our basic ping works.
+
+![Basic Ping](basic_ping.png)
+
+As you go through the rest of the Networking and Policy Tutorials, you can keep experimenting with different queries and viewing graphs on Grafana.
+
+### <a name="resources"></a> Resources
+
+For more query functions, find them [here](https://prometheus.io/docs/querying/functions/).
+
+### Improvements or Comments
+This tutorial was developed by Contiv engineers. Thank you for trying out this tutorial.
+Please file a GitHub issue if you see an issue with the tutorial, or if you prefer
+improving some text, feel free to send a pull request.
```

Comment:
^ this is a section that we put on all the tutorials, so i just copied and pasted that. should I add the `CONTRIBUTING.md` to all of the tutorials?